config/environment.js

```diff
@@ -10,18 +10,18 @@ module.exports = function(environment) {
       FEATURES: {
         // Here you can enable experimental features on an ember canary build
         // e.g. EMBER_MODULE_UNIFICATION: true
-        EMBER_METAL_TRACKED_PROPERTIES: true
+        EMBER_METAL_TRACKED_PROPERTIES: true,
```

Comment:
It is not in a default Ember app for version 3.28, because it seems related to migration to Octane. No idea what it really means. Should we just remove this?

Reply:
It's not in the 3.9 blueprint either. Those flags are only used if you use the "canary" version of Ember, so not the built / pushed to npm ones.

Yea we should just remove it, but it doesn't matter that much. I just noticed it while scrolling through the changes 😄.